### ISSUE TITLE
Broken rendering on games which use dual vertex shaders

### ISSUE THREAD

**Issue by winderica** [yuzu-emu/yuzu#3439]
The game Higurashi No Naku Koro Ni Hou doesn't render. Instead, it just gives a black screen.

* Yuzu version: 171 ([commit 450a4bf62](https://github.com/yuzu-emu/yuzu-mainline/commit/450a4bf624910c1e879c06bcf31cb95eb1e4dbc7)), and version 170 produces the same result
* OS: Windows 10 19564.1005
* CPU: i7-7700HQ
* GPU: GTX 1050 Ti

Screenshot: 

![screenshot](https://user-images.githubusercontent.com/13536040/74944251-b6c8d580-5430-11ea-877d-cf2d682f730f.png)

Log file: [yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/4231242/yuzu_log.txt)

I am not sure whether these issues (#2056, #2626) are related to this problem, but it seems that they have already been fixed.

**Comment by FearlessTobi** [comment]
Likely cause could be this:
```[  16.522982] Render.OpenGL <Error> video_core/renderer_opengl/gl_shader_util.h:LoadProgram:79: Error linking shader:
Vertex info
-----------
utof(165) : error C7607: Redeclaration of gl_Position must precede its usage
utof(165) : error C1038: declaration of "gl_Position" conflicts with previous declaration at 0(31)
(0) : error C2003: incompatible options for link

**Comment by Hexagon12** [comment]
This is due to the game using a dual vertex shader setup (VertexA and VertexB), with the latter one currently being broken.

* Happens in Farming Simulator

**Comment by Morph1984** [comment]
Fixed by #6585
